### ISSUE TITLE
Cloud Init Physical Install Issues

### ISSUE THREAD

**Issue by crawford** [coreos/bugs#730]
<a href="https://github.com/maxexcloo"><img src="https://avatars.githubusercontent.com/u/927210?v=3" align="left" width="96" height="96" hspace="10"></img></a> **Issue by [maxexcloo](https://github.com/maxexcloo)**
_Saturday May 10, 2014 at 07:06 GMT_
_Originally opened as https://github.com/coreos/coreos-cloudinit/issues/102_

----

Hello!
I have an issue open over on the CoreOS install repo and figured it may be worth raising it here as it may be relevant.
https://github.com/coreos/init/issues/77

Basically I'm trying to install CoreOS Beta to disk from the CoreOS ISO using a cloud init file (http://pastie.org/pastes/9161456/text). The config file gets written to /var/lib/coreos-install/user_data and everything seems fine.

However, on reboot my hostname doesn't seem to be used and I cannot log in with my SSH Keys.

Any help would be appreciated, thank you!

**Comment by crawford** [comment]
<a href="https://github.com/marineam"><img src="https://avatars.githubusercontent.com/u/153640?v=3" align="left" width="48" height="48" hspace="10"></img></a> **Comment by [marineam](https://github.com/marineam)**
_Saturday May 10, 2014 at 07:18 GMT_

----

Closing to keep the conversation in the original bug.
